packit/cli/push_update.py

```diff
@@ -0,0 +1,51 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+Push the update/updates to stable.
+"""
+
+import logging
+from os import getcwd
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception
+from packit.cli.utils import get_packit_api
+from packit.config import pass_config, get_context_settings
+
+logger = logging.getLogger(__file__)
+
+
+@click.command("push-update", context_settings=get_context_settings())
+@click.argument("update_alias", default=None)
+@click.argument("path_or_url", type=LocalProjectParameter(), default=getcwd())
+@pass_config
+@cover_packit_exception
+def push_update(update_alias, config, path_or_url):
+    """
+    Push the update/updates to stable.
+
+    """
+    api = get_packit_api(config=config, local_project=path_or_url)
+    api.push_updates(alias=update_alias)
```

Comment:
Just a detail:  `push_update` (cli) vs. `push_updates` (api)
Shouldn't it be unified?